c++/include/orc/Statistics.hh

```diff
@@ -37,12 +37,76 @@ namespace orc {
      * of rows because of NULL values and repeated values.
      * @return the number of values
      */
-    virtual uint64_t getNumberOfValues() const = 0;
+    virtual uint64_t getNumberOfValues() const {
+      return valueCount;
+    }
+
+    /**
+     * Set the number of values in this column
+     * @param newValueCount new number of values to be set
+     */
+    virtual void setNumberOfValues(uint64_t newValueCount) {
```

Comment:
If we put this into Impl classes, then there are many duplicate code which is not elegant. So we decided to lift them up to the base classes so that duplicate codes are reduced significantly while we were implementing the writer. What do you think?